### PR TITLE
Continuation of #8561 - add windows.service.start_name and windows.service.path_name

### PR DESCRIPTION
**Pull request by narph**
Fixing conflicts for elastic/beats#8561 in order to get the PR in.

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -1332,8 +1332,11 @@ https://github.com/elastic/beats/compare/v6.4.0...v6.5.0[View commits]
 - Precalculate composed id fields for kafka dashboards. {pull}8504[8504]
 - Add support for `full` status page output for php-fpm module as a separate metricset called `process`. {pull}8394[8394]
 - Add Kafka dashboard. {pull}8457[8457]
+- Added `path_name` and `start_name` to service metricset on windows module {issue}8364[8364]
+- Add untyped metric support to the prometheus module. {pull}8681[8681]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: Rebase issue?

Comment (narph) [reply CHANGELOG.asciidoc]: yes

**Comment by narph** [comment]
jenkins, test this

**Comment by narph** [comment]
jenkins, test this

**Pull request merged**